emails/apps.py

```diff
@@ -20,6 +20,8 @@ def __init__(self, app_name, app_module):
                 'ses', region_name=settings.AWS_REGION
             )
             self.s3_client = boto3.client('s3', region_name=settings.AWS_REGION)
+            self.sqs_client = boto3.resource('sqs', region_name=settings.AWS_REGION)
+            self.dl_queue = self.sqs_client.Queue(settings.AWS_SQS_QUEUE_URL)
```

Comment:
@groovecoder do you know if an SQS auth attempt is done here? If so maybe we should have a separate try/except section? Otherwise I think it would always log as `exception during SES connect`

Reply:
Moved this into the command itself, as @say-yawn suggested. It will now log as `sqs_client_error`.